src/Dialogs/Settings/Panels/GaugesConfigPanel.cpp

```diff
@@ -122,8 +124,20 @@ GaugesConfigPanel::Prepare(ContainerWindow &parent,
   AddBoolean(_("Vario bar"),
              _("If set to ON the vario bar will be shown"),
              map_settings.vario_bar_enabled);
-
   SetExpertRow(EnableVarioBar);
+
+  AddSpacer();
+  SetExpertRow(SPACER);
+
+  AddInteger(
+    _("Navigator Height"),
+    _("Select the height of the navigator topwidget (percentage of the main window).\n\n"
+      "This widget is set in the System Menu --> Look --> Pages --> Top Area\n\n"
```

Comment:
ASCII art replaced ('/' instead of "-->") and all double '\n' removed:
"Select the height of the navigator topwidget (percentage of the main window).\n"
"This widget is set under the settings menu: System / Look / Pages / Top Area\n"

OK for you?